jenkins_home/credentials.xml

```diff
@@ -55,6 +55,31 @@
           <description>Jenkins Github Shared Secret</description>
           <secret>{AQAAABAAAAAwTKFWtX7H1OC8M7/0JeYfWxfb3w779gD1rynbbkNLhjZwnXP1A9A1mrOs/IiSiAzDDVJJAOg0qbcV9iS0DfXlnQ==}</secret>
         </org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl>
+        <org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl plugin="plain-credentials@1.4">
+          <scope>GLOBAL</scope>
+          <id>PILOT_CODECOV_TOKEN</id>
+          <description>Codecov for Pilot</description>
+          <secret>{AQAAABAAAAAwxrwEn9J2ro5v26Pb/gIZy/kPZPCxL2ChxGot2JJ8ENSfF5e3gJw16Ltw+oc9wT8wjDS8fMYSClHwVi2ptKJyvw==}</secret>
```

Comment:
I might be missing something, but I don't think it is a good idea to store secrets in clear text and be shared on github.